addons/crm/models/crm_lead.py

```diff
@@ -1254,10 +1255,12 @@ def get_empty_list_help(self, help_message):
             ('alias_parent_model_id.model', '=', 'crm.team'),
             ('alias_force_thread_id', '=', False)
         ], limit=1)
+
         if alias_record and alias_record.alias_domain and alias_record.alias_name:
-            email = '%s@%s' % (alias_record.alias_name, alias_record.alias_domain)
-            email_link = "<b><a href='mailto:%s'>%s</a></b>" % (email, email)
-            sub_title = _('Use the top left <i>Create</i> button, or send an email to %s to test the email gateway.') % (email_link)
+            email = f'{alias_record.alias_name}@{alias_record.alias_domain}'
+            sub_title = Markup(_('Use the top left <i>Create</i> button, or send an email to %(email_link)s to test the email gateway.')) % {
+                'email_link': Markup("<b><a href='mailto:%s'>%s</a></b>") % (email, email),
+            }
```

Comment:
We can call Markuponly once
```suggestion
            sub_title = Markup(_('Use the top left <i>Create</i> button, or send an email to <b><a href='mailto:%s'>%s</a></b> to test the email gateway.')) % (email, email)
```

Reply:
Purpose is to get the href out of the string to translate actually :) that way we lessen potential issues if people wrongly translate html. Few HTML is ok but when possible ...

Reply:
ah ok :) ok for me so :)